.kube/gke-deployment.yml

```diff
@@ -53,21 +53,19 @@ spec:
         tier: backend
         version: 1.0.0
     spec:
+      volumes:
+      - name: appsettings-config
+        configMap:
+          name: app-config
       containers:
       - name: webapi-api
         image: gcr.io/agrc-204220/webapi/api
         imagePullPolicy: Always
+        volumeMounts:
+        - name: appsettings-config
+          mountPath: /app/appsettings.json
+          subPath: appsettings.json
         env:
-        - name: WEBAPI__DATABASE__HOST
-          value: db
-        - name: WEBAPI__DATABASE__PORT
-          value: '5432'
-        - name: WEBAPI__DATABASE__DB
-          value: webapi
-        - name: WEBAPI__DATABASE__USERNAME
-          value: postgres
-        - name: WEBAPI__DATABASE__PASSWORD
-          value: what password
         - name: WEBAPI__DATABASE__PEPPER
```

Comment:
let's move the pepper value as well so all of the webapi:database things are in the same place